deploy/hackathon-docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is covered by the LICENSING file in the root of this project.
+#
+
+### Dockerfile for open-hacnathon(server and client)
```

Comment:
Suggest deploy client and server separatedly

Reply:
how about we merge this as step 1, i will separate client as a dedicated one after that, what's more, add guacamole in docker compose as well.

Reply:
a lot of "localhost" in config_sample need to be taken care of.

Reply:
You need another config file like config_docker.py or similary and read from ENV variables which are defined in Dockerfile and passed by by docker comand

Reply:
this is my way and it works, what do you think:

if "DB_SERVER" in os.environ:
    MONGODB_HOST = os.environ["DB_SERVER"]
else:
    MONGODB_HOST = "localhost"

if "DB_PORT" in os.environ:
    MONGODB_PORT = int(os.environ["DB_PORT"])
else:
    MONGODB_PORT = 27017

Reply:
Not those kind of variables. What I meant to refer to are those cannot be submitted to repo such as Github OAuth Client Secerts, which you cannot put it in any Dockerfile/py file.